lib/tasks/db.rake

```diff
@@ -23,4 +23,14 @@ namespace :db do
       SetAssetSizeWorker.perform_async(asset_id)
     end
   end
+
+  desc "Transform all redirect chains into one-step redirects"
+  task resolve_redirect_chains: :environment do
```

Comment:
Is the intention for this task to be run once and then deleted?

Reply:
Yes